components/infieldbutton/metadata/infieldbutton.yml

```diff
@@ -109,7 +112,7 @@ examples:
     markup: |
```

Comment:
Just curious what the Workflow icon only variant is displaying that is different than the other variants and if it could benefit from a description?

(not sure if this is the right line 😅)

Reply:
I'm honestly not sure. It was a separate option from the others so I left it there? I know we have two different types of icons but since we're already using a workflow icon maybe it should show with a UI icon only?